### ISSUE TITLE
keyboard shortcuts intercepted on experiment window

### ISSUE THREAD

**Issue by carandraug** [MicronOxford/cockpit#571]
The global keyboard shortcuts work on multiple windows, even if that window has nothing to do with the action for the keyboard shortcut. I imaging some people want that? Even if that is desirable, on the single site experiment window, the keyboards work but some shouldn't because they can be used on the actual UI. To replicate:

1. open the single site experiment window
2. select the "Experiment type" choice menu
3. press up and down to try and change the experiment type
4. note that instead the z stage moves up and down

I'm not sure what other keypress may get intercepted since my keyboard does not have a numpad but I wonder if pressing a 2 on the keypad to set the Z stack height on the experiment window will instead move the Y axis.

**Comment by iandobbie** [comment]
Yes the key presses will go to the motion command and not enter text
